src/puppetlabs/puppetdb/http/handlers.clj

```diff
@@ -127,6 +127,17 @@
                        identity
                        (http/status-not-found-response "environment" environment)))))
 
+(defn producer-status
+  "Produce a response body for a single producer."
```

Comment:
Again, that was following the format of environment-status, above. 
Let me know if you still think I should change it.